app/frontend/components/pages/delegations/stakePoolInfo.tsx

```diff
@@ -4,12 +4,26 @@ import {Stakepool, Lovelace} from '../../../types'
 import {h} from 'preact'
 import tooltip from '../../common/tooltip'
 
+// REFACTOR: (Untyped errors): move to types
+// is "hasTickerMapping" something specific or general?
+type Error = {
+  code: string
+  params?: {hasTickerMapping: boolean}
+}
+
 type StakePoolInfoProps = {
   pool?: Stakepool
   gettingPoolInfo: boolean
+  validationError: Error
+  empty: boolean
 }
 
-export const StakePoolInfo = ({pool, gettingPoolInfo}: StakePoolInfoProps): h.JSX.Element => {
+export const StakePoolInfo = ({
+  pool,
+  gettingPoolInfo,
+  validationError,
+  empty,
```

Comment:
It seems to me we don't need this parameter, passing pool as null should imho solve the issue and also keep the current behaviour.